### ISSUE TITLE
Undefined references in ns3 libs when running loadshed example

### ISSUE THREAD

**Issue by czekster** [pnnl/tesp#24]
/usr/bin/ld: /home/merlin/tesp/lib/libns3-dev-internet-debug.so: undefined reference to `typeinfo for ns3::Probe'
usr/bin/ld: /home/merlin/tesp/lib/libns3-dev-internet-debug.so: undefined reference to `ns3::Probe::~Probe()'
/usr/bin/ld: /home/merlin/tesp/lib/libns3-dev-internet-debug.so: undefined reference to `ns3::PacketFilter::~PacketFilter()'
/usr/bin/ld: /home/merlin/tesp/lib/libns3-dev-point-to-point-debug.so: undefined reference to `vtable for ns3::MpiReceiver'
/usr/bin/ld: /home/merlin/tesp/lib/libns3-dev-internet-debug.so: undefined reference to `ns3::TrafficControlHelper::Install(ns3::Ptr<ns3::NetDevice>)'

The libs are in the TESP_INSTALL/lib folder

**Comment by temcdrm** [comment]
We install a file called tesp_ld.conf under /etc/ld.so.conf.d.  Can you please edit that and replace /opt/tesp/lib with /home/merlin/tesp/lib in your case? Then "sudo ldconfig". If that works, we can probably fix the installer to handle it.

tom@uvm:~/src/tesp/install/Linux/helpers$ cat **tesp_ld.conf**
# put this in /etc/ld.so.conf.d
**/opt/tesp/lib**

**Comment by czekster** [comment]
Ok, it compiled.
The problems are happening because I haven't installed TESP on /opt/tesp
(I'll remove older installations and install on /opt/tesp)
It could be related, but while trying to run loadshed it yields:

loadshed.glm(66): class 'helics_msg' is not known
loadshed.glm(66): load failed at or near 'object helic...'
FATAL    [INIT] : shutdown after command line rejected
Traceback (most recent call last):
  File "helicshed.py", line 2, in <module>
    import helics as h
ModuleNotFoundError: No module named 'helics'

**Comment by temcdrm** [comment]
That means another version of GridLAB-D was found first, i.e., one that wasn't linked with HELICS.  If you have two versions of GridLAB-D, please edit the PATH and GLPATH environment variables as suggested in the new instructions here: https://tesp.readthedocs.io/en/latest/Installing_Ubuntu_Link.html

`If you have already installed GridLAB-D, EnergyPlus, ns-3 or another simulator before TESP, it may be necessary to manually adjust your PATH and GLPATH environment variables when running TESP. The TESP versions of these simulators are built with HELICS and FNCS interfaces, which are not normally included in other versions of the simulators. In GridLAB-D, you may get an error message about “module connection” not being found. With other simulators, you might see the FNCS or HELICS broker hang, waiting for a federate that’s unable to connect.`

**Comment by czekster** [comment]
1. Downloaded version 0.6.3 from GitHub;
2. Set $GLPATH to /opt/tesp/bin and $TESP_INSTALL to /opt/tesp and TESPDIR (same);
3. Set $PATH to find first tesp's dir then the rest, e.g., `export PATH=/opt/tesp/dir:$PATH;`
4. Ran a `gridlabd --version` and it came the same version as the one in /opt/tesp/dir 

Then I tried to run the comm example and the same error occurred for when the module 'metrics' is being loaded in the .glm file.

For the loadshed example, it says that class 'helics_msg' is not known. 

For some reason, it's still not finding the right gridlabd.
Is there any other variable needed to be set?
